### PR TITLE
[TA] Prepare for November Release.

### PR DESCRIPTION
**Pull request by mssfang**
- Prepare for the November release

### CODE REVIEW

File: sdk/textanalytics/tests.yml [hunk 1]
```diff
@@ -8,9 +8,5 @@ jobs:
         - name: azure-ai-textanalytics
           groupId: com.azure
           safeName: azureaitextanalytics
-      # resource creation is disabled because we are trying to test the stable endpoint.
-      # Will enable it when service rolls out AAD to all region.
-      DisableAzureResourceCreation: true
       EnvVars:
-        AZURE_TEXT_ANALYTICS_API_KEY: $(java-textanalytics-test-api-key)
-        AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-textanalytics-sdk-endpoint)
+        AZURE_TEST_MODE: RECORD
```
Comment (maririos) [inline sdk/textanalytics/tests.yml:12]: Will this allow the live test to run?

Comment (mssfang) [reply sdk/textanalytics/tests.yml:12]: Yes

**Review by samvaity — APPROVED** [review]

**Review by maririos — APPROVED** [review]

**Pull request merged**